apps/src/turtle/skins.js

```diff
@@ -165,8 +165,41 @@ exports.load = function(assetUrl, id) {
 
     return mapping;
   };
+  /**
+   * Generates a mapping of geometry sticker names to the urls of their images.
+   *
+   * @return the mapping of names to urls
+   */
+  var patternBlocks = function() {
+    // Pattern Blocks
+    var patternBlocks = [
```

Comment:
I missed this on the other PR, but I wonder if the names on the rhombuses could reflect their color, rather than number, i.e. 'smallRhombusRed'. That would help make sure the names are descriptive and that if we ever get feedback like 'the yellow rhombus clashes with the logo, can we remove', it would be easy to quickly identify which files they are.

Reply:
Got it. If these are just placeholders, the naming is fine as is.